packages/kotti-inline-edit/src/InlineEdit.vue

```diff
@@ -86,10 +86,10 @@ export default {
 		},
 	},
 	methods: {
-		handleConfirm(event) {
+		handleConfirm() {
 			this.editMode = false
 			this.preValue = this.currentValue
-			this.$emit('confirm', event)
+			this.$emit('confirm', this.currentValue)
```

Comment:
also it seems that we sync currentValue with value so it shouldn't be problematic
but if we change confirm then should we consider also changing the dismiss for consistency ?

Reply:
this.$emit('confirm', {...event, currentValue: this.currentValue }) would keep the event data being available and have the current value available.